### PR TITLE
Fast-retrieval copy plumbing

### PR DESCRIPTION
**Pull request by magik6k**

### CODE REVIEW

File: types.go [hunk 1]
```diff
@@ -141,6 +142,30 @@ func (t *SectorInfo) sealingCtx(ctx context.Context) context.Context {
 	return ctx
 }
 
+func (t *SectorInfo) unsealedRanges(invert bool) []storage.Range {
```
Comment (whyrusleeping) [inline types.go]: this function (and especially its parameter) are kinda confusing. What is being inverted? its not a super intuitive option.

**Review by arajasek — APPROVED** [review]
LGTM

File: types.go (hunk 1 above)
Comment (arajasek) [reply types.go]: Agreed, but it makes sense if you squint at it. I think it's fine with a one-line explainer on top.

**Review by whyrusleeping — APPROVED** [review]
seems like its doing the right thing, but the things are confusingly named

**Pull request merged**